### DISCUSSION TITLE
deos `extract_words()`have the argument use_text_flow?

### DISCUSSION THREAD

**Discussion by faris404** [jsvine/pdfplumber#558] [Q&A] [answered]
I am getting this error when the below code runs
```py
page.extract_words(
                    use_text_flow=True,
                    y_tolerance=0.5,
                    keep_blank_chars=True)
```
error

```TypeError: extract_words() got an unexpected keyword argument 'use_text_flow```

version
```
pdfplumber==0.5.23
```
Does extract_words() have the argument use_text_flow (it's in the docs)?
If yes why I am getting this error?

Thank you

**Comment by samkit-jain** [comment, marked as answer]
Hi @faris404 Appreciate your interest in the library. The param was added in pdfplumber v0.5.24 and should be available on all the versions released after it including 0.5.24. Could you please confirm what version of the library are you using? If it is an older version, request you to upgrade to the supported ones. If not, please share a fully reproducible example along with the PDF for us to investigate further.

**Reply by faris404** [reply]
It works. Thank you for your time
